gcc/rust/util/rust-mapping-common.h

```diff
@@ -62,8 +62,8 @@ struct DefId
 };
 
 #define UNKNOWN_CRATENUM ((uint32_t) (UINT32_MAX))
-#define UNKNOWN_NODEID ((uint32_t) (0))
-#define UNKNOWN_HIRID ((uint32_t) (0))
+#define UNKNOWN_NODEID ((uint32_t) (UINT32_MAX))
+#define UNKNOWN_HIRID ((uint32_t) (UINT32_MAX))
```

Comment:
These should not be 0.
0 is used for an existing node. (I wasted time on this...🙃)